src/earth/earth.c

```diff
@@ -546,7 +549,14 @@ earthPreparePaintScreen (CompScreen *s,
 	es->gha = (float)currenttime->tm_hour-(es->tz +
 				(float)currenttime->tm_isdst) +
 				(float)currenttime->tm_min / 60.0f;
-
+				(float)currenttime->tm_sec/3600.0000f;
```

Comment:
This statement does nothing. It was probably meant to replace `(float)currenttime->tm_min / 60.0f;` with `(float)currenttime->tm_sec/3600.0000f;`